resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/SparkKubernetesClientFactory.scala

```diff
@@ -102,6 +110,8 @@ private[spark] object SparkKubernetesClientFactory extends Logging {
     val httpClientWithCustomDispatcher = baseHttpClient.newBuilder()
       .dispatcher(dispatcher)
       .build()
+    logDebug("Kubernetes client config: " +
```

Comment:
nit: I think the log can be moved before L109(`HttpClientUtils.createHttpClient(config)`)

Reply:
I would like to keep it as close to `DefaultKubernetesClient` as possible.

Reply:
As that is the one which we are mostly interested in.